### ISSUE TITLE
The menu is broken on a tablet on ICS (Android 4)

### ISSUE THREAD

**Issue by tbassetto** [joshfire-platform-templates/event#9]
Each items is on the same line (good) but the "three dots" button is still visible and clicking on it will show the second line … empty.

**Comment by steren** [comment]
I can understand that the Phone UI is loaded, but didn't you expect the tablet one for this device?

Anyway, I agree, the "more" button is added even if it is not needed.

**Comment by tbassetto** [comment]
Right, the main problem is that the tablet layout should have been used.
